File: src/webapp/src/model/question.ts
export class Question {

  id: number;
  label: string;
  fieldType: string;
  required: boolean;
  options: string[];
  questionType: string;
  description: string;
  position: number;

  constructor(questionType: string) {
    this.options = [];
    this.fieldType = 'TEXT';
    this.required = false;
    this.questionType = questionType;
  }
}

File: src/main/java/app/oengus/domain/marathon/Question.java
package app.oengus.domain.marathon;

import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@RequiredArgsConstructor
public class Question {
    private final int id;
    private String marathonId;

    private String label;
    private boolean required;
    private List<String> options;
    private FieldType fieldType;
    private QuestionType type;
    private String description;
    private int position;
}
